helm/handbook/values.schema.json

```diff
@@ -5,16 +5,27 @@
         "staticJsCmsHugoStandaloneVersion": {
             "type": "string"
         },
-        "secrets": {
+        "Secrets": {
```

Comment:
Why the case change here? YAML properties should normally be camel cased, not classic

Reply:
It's done like this in other projects. I looked at `area-oncall-scheduler`.

It was me who converted it to camel-case initially and I reverted that later, because capital `Secrets` seemed like a convention in values.yaml.

What is the definitive answer?